Sources/MongoSwift/Options.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
```

Comment:
kind of a lame name for this file but it seemed like this stuff should be grouped together somewhere... so let me know if you have any opinions